### ISSUE TITLE
Wrong port in admin interface

### ISSUE THREAD

**Issue by mkevac** [centrifugal/centrifugo#150]
Admin interface gives me wrong addresses. With wrong ports.

<img width="666" alt="upload" src="https://cloud.githubusercontent.com/assets/123544/22782802/e0da4756-eed9-11e6-856c-965daebf4546.png">

Even though centrifugo is listening on 12000:

```
[3:00 PM] Marko Kevac: [I]: 2017/02/09 11:47:05 Start serving raw websocket, SockJS, API endpoints on :12000
[3:00 PM] Marko Kevac: [I]: 2017/02/09 11:47:05 Start serving admin endpoints on :13000
```

Centrifugo was started like this:
```
docker run -d --name=centrifugo --ulimit nofile=65536:65536 -v /local/centrifugo/conf:/centrifugo --network=host centrifugo/centrifugo centrifugo -c config.json --ssl --ssl_cert=server.crt --ssl_key=server.key -p 12000 --admin --web --admin_port=13000
```

**Comment by FZambia** [comment]
@mkevac thanks for report! That's because web interface tries to show those endpoints [based on browser url](https://github.com/centrifugal/web/blob/master/app/src/js/app.jsx#L109) - before adding different ports for different handlers options (`--admin_port` and `--api_port`) it worked pretty good, but now we have to find a new way to deal with it.

**Comment by FZambia** [comment]
Looks like the best solution will be just remove those lines from web interface as they can be confusing sometimes and better document those endpoint addresses. 

Even if rely on configuration options coming from Centrifugo we won't be able to show correct endpoints as depending on proxy/load balancer setup connection/api domain addresses can differ from admin web interface address. Also we can't say exactly should we use ports from config in these lines or omit them because Centrifugo behind Nginx for example.

**Comment by FZambia** [comment]
Added [more docs](https://fzambia.gitbooks.io/centrifugal/content/server/advanced_configuration.html#advanced-endpoint-configuration) about endpoints and port configuration.

**Comment by FZambia** [comment]
@mkevac just released v1.6.5 with changes - there are no connection endpoints shown anymore.
